frontend/src/metabase-types/store/embed.ts

```diff
@@ -1,4 +1,4 @@
-export interface EmbedOptions {
+export interface InteractiveEmbeddingOptions {
```

Comment:
The gist is that we have so many kinds of embeddings, and this type is specifically for interactive embedding, so it's clearer to name it like so.